src/test/java/ch/interlis/iox_j/validator/LogCollectorAssertions.java

```diff
@@ -0,0 +1,30 @@
+package ch.interlis.iox_j.validator;
+
+import ch.interlis.iox.IoxLogEvent;
+import org.junit.Assert;
+
+import java.util.List;
+
+public class LogCollectorAssertions {
+    public static void AssertContainsError(String text, int count, LogCollector logCollector){
+        Assert.assertEquals( String.format("Wrong number of error logs containing <%s> found.", text), count, CountOccurrences(text, logCollector.getErrs()));
```

Comment:
PP: In der Assert message den Count auch noch einbauen.

Reply:
Wird bereits in der Message von `assertEquals` ausgegeben.